client/blocks/edit-gravatar/test/index.jsx

```diff
@@ -36,6 +36,7 @@ jest.mock( 'calypso/components/file-picker', () =>
 		</div>
 	) )
 );
+jest.mock( 'calypso/state/selectors/get-user-settings', () => jest.fn( () => {} ) );
```

Comment:
This simply fixes unit tests. Edit-gravatar uses the dialog which was not previously connected to user settings. Since the user settings state pulls settings off of nested objects, just the state connection of the component caused this test to fail on accessing properties of undefined. This bypasses that error so the edit-gravatar tests can run as expected.